test/sascalculator/utest_sas_gen.py

```diff
@@ -251,7 +258,7 @@ def test_debye_impl(self):
             errs = (external - analytical)/analytical
             different_entries = 0
             for val in np.abs(errs):
-                self.assertLessEqual(val, 0.02, "Ensure that the error is acceptable.")
+                self.assertLessEqual(val, 0.03, "Ensure that the error is acceptable.")
                 if val != 0:
```

Comment:
Given how floating-point comparisons have a habit of kicking one in the backside... I'd expect that this would often be just slightly off zero. Is there a closeness threshold that makes sense instead?